akinet/net_traffic.go

```diff
@@ -374,6 +374,30 @@ func (tls *TLSHandshakeMetadata) ApplicationLatencyMeasurable() bool {
 	return *tls.SelectedProtocol == "http/1.1"
 }
 
+// Represents the content when a TCP connection is closed by the client.
+type ClientTimeoutMetadata struct {
```

Comment:
This looks OK, but it might be better to specifically say "Shutdown" rather than "Timeout" to be more specific.  (We don't _know_ that it timed out -- it might have crashed or been killed.  So timeout is an inference that might not be justified.)